### ISSUE TITLE
What's the relation between batch_sampler and real batchsize?

### ISSUE THREAD

**Issue by LingyvKong** [ModelTC/United-Perception#51]
Hello, If I train a network with 2 gpus, the real batchsize is batch_sampler? or 2*batch_sampler?
what about the learning rate?

Thank you

**Comment by yqyao** [comment]
the total batchsize = batchsize * gpu_num @LingyvKong
